patches/minecraft/net/minecraft/world/level/storage/loot/LootContext.java.patch

```diff
@@ -66,3 +66,14 @@
              }
           }
        }
+@@ -230,6 +_,10 @@
+ 
+       public LootContextParam<? extends Entity> m_79003_() {
+          return this.f_78995_;
++      }
++
++      public String getName() {
```

Comment:
I'd like for this patched-in method to be clearly marked as such with a comment (javadoc or regular).

Reply:
I concur. This can be very misleading during a port update.